var/spack/repos/builtin/packages/htslib/package.py

```diff
@@ -49,11 +49,16 @@ class Htslib(AutotoolsPackage):
         default=True,
         description="use libdeflate for faster crc and deflate algorithms",
     )
+    variant("gcs", default=True, description="enable gcs url support", when="+libcurl@1.5")
+    variant("s3", default=True, description="enable s3 url support", when="+libcurl@1.5")
```

Comment:
A suggestion for readability:
```suggestion
    variant("gcs", default=True, description="enable gcs url support", when="@1.5+libcurl")
    variant("s3", default=True, description="enable s3 url support", when="@1.5+libcurl")
```
Also, more importantly, why this variant is active only for version `@1.5` ? Is it worth it?

Reply:
Good catch. Thank you!

Reply:
This should be `@1.5:` instead of just `1.5`. Prior to that (i.e., in `@1.3:1.4`), they were always built but they're independently configurable from 1.5 onwards.